education/blockchain/mining-followup.md

```diff
@@ -0,0 +1,3 @@
+# Proof of Work
```

Comment:
When looking at the content flow diagram that we created, I imaged the breakdown (when it comes to file structure of the education section) would look like this.

Each topic (depth level 2) has it's own file. The file has a structure as follows:

```
# Header (the name of the topic usually)
<A paragraph or two overview of the topic>
```
This is the (shallow exposure) to the topic. For example: "Hashing is like a fingerprint". Boom, done. That is all anyone who is curious about it needs to know. If they want to know more though, the proceed to the subsections which are designated by H2 headers.

In mining this would be like: 

```
## Proof of Stake
Full explanation of Proof of Stake with analogies

## Proof of Work
Full explanation of Proof of Work with analogies
```

This was the setup I was establishing beforehand.